typify-impl/src/merge.rs

```diff
@@ -77,12 +81,30 @@ fn merge_schema(a: &Schema, b: &Schema, defs: &BTreeMap<RefKey, Schema>) -> Sche
     try_merge_schema(a, b, defs).unwrap_or(Schema::Bool(false))
 }
 
+#[derive(Error, Debug)]
+pub(crate) enum SchemaMergeError {
+    #[error("Cannot merge two trivially false schemas")]
```

Comment:
We may be merging two `false` schemas... or just one. I'd suggest:

```suggestion
    #[error("merging with false")]
```

Or nothing at all if there's no additional information to be provided.

Reply:
Done